src/frontend/App/context/BehandlingContext.tsx

```diff
@@ -43,6 +45,15 @@ const [BehandlingProvider, useBehandling] = constate(() => {
             ),
         [behandling]
     );
+    useEffect(() => {
+        if (behandlingErRedigerbar) {
+            axiosRequest<string | null, string>({
+                method: 'GET',
+                url: `/familie-ef-sak/api/oppgave/${behandlingId}/tilordnet-ressurs?saksbehandlerIdent=${innloggetSaksbehandler.navIdent}`,
```

Comment:
Hvem som gjør requesten har vi i token, så burde være tilstrekkelig å kun kalle endepunkten uten å sende med navIdent? 
```suggestion
                url: `/familie-ef-sak/api/oppgave/${behandlingId}/tilordnet-ressurs`,
```

Reply:
Jeg fjernet behovet for request param `saksbehandlerIdent` når jeg la til tidspunkt i loggene

Reply:
fixd i [6efd4da](https://github.com/navikt/familie-ef-sak-frontend/pull/1458/commits/6efd4da5a5e23edc66e1900e961542f82ca2862c) måtte fjerne en import også :)